client-src/entities/Player.ts

```diff
@@ -199,9 +212,11 @@ export class Player extends MatterContainer {
     }
 
     onTouchingItemStart(myBody: any, itemBody: any, activeContacts: IMatterContactPoints) {
+        log('onTouchingItemStart', myBody.isSensor, myBody.label, this.pointerTarget?.name);
         // a and b are bodies, but no TS definition...
         if (!itemBody.isSensor) return;
         if (myBody.label !== 'hand') return;
+        console.log('onTouchingItemStart do');
```

Comment:
should use log

Reply:
that log is still too verbose. i should comment it out rather than make it permanent